### DISCUSSION TITLE
Securing Ubuntu vps of wireguard

### DISCUSSION THREAD

**Discussion by KourosZanbouri** [angristan/wireguard-install#415] [Help]
Hello everybody,

I have a question, Do i need to secure my VPS ( ubuntu OS) that runs wireguard/openvpn services ( both used @angristan scripts ) to prevent attacks such as DoS, DDoS or ... to my server ? if Yes, what do i need? ( cdn ? basic firewall ? ... ? )

ps. the VPS is using only wireguard and OpenVPN without any other services...

**Comment by ChefG0rd0n** [comment]
I think it should be best practice to secure VPS's from all potential avenues of attack. That includes editing the SSH config file, and whitelisting IP access 👍
